pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: protoc_plugin
-version: 0.7.6
+version: 0.7.7-dev
```

Comment:
This is a nice convention – only remove the `-dev` when you're ready to release.